packages/sdk/src/views/streams/timelinesModel.ts

```diff
@@ -347,6 +348,21 @@ export function makeTimelinesViewInterface(
         streamId: string,
         updatingEventId?: string,
     ) {
+        // if it was tagged as a reaction but the content is not a reaction, just ignore
+        // but don't filter out encrypted events as they may decrypt to valid reactions
+        if (
+            isReactionInteraction(event.tags) &&
+            event.content?.kind !== RiverTimelineEvent.Reaction &&
+            event.content?.kind !== RiverTimelineEvent.ChannelMessageEncrypted &&
+            event.content?.kind !== RiverTimelineEvent.ChannelMessageEncryptedWithRef
+        ) {
```

Comment:
i think we may need to flip this around, if it's a top level POST that's not tagged as a POST then it needs to be ignored..